lib/normalize.js

```diff
@@ -67,31 +67,59 @@ const normalizeSources = dp => {
   for (const idx in dp.sources) {
     dp.sources[idx].title = dp.sources[idx].name
   }
-  for (const idx in dp.resources){
-    for (const id in dp.resources[idx].sources){
-      if(dp.resources[idx].sources[id].title) {
+  for (const idx in dp.resources) {
+    for (const id in dp.resources[idx].sources) {
+      if (dp.resources[idx].sources[id].title) {
         dp.resources[idx].sources[id].title = dp.resources[idx].sources[id].title
       }
-      else if(dp.resources[idx].sources[id].name){
+      else if (dp.resources[idx].sources[id].name) {
         dp.resources[idx].sources[id].title = dp.resources[idx].sources[id].name
       }
       else {
-        dp.resources[idx].sources[id].title = 'unkown'
+        dp.resources[idx].sources[id].title = 'unknown'
       }
     }
   }
   return dp
 }
 
 const normalizeContributors = dp => {
-  for (const idx in dp.contributors){
-    dp.contributors[idx].name = dp.contributors[idx].name.toLowerCase().replace(/\s/g, '_')
+  const normalizeContirutorsTitle = dp => {
+    for (const idx in dp.contributors) {
+      if (dp.contributors[idx].title) {
+        dp.contributors[idx].title = dp.contributors[idx].title
```

Comment:
This is pointless - you're checking if it has a title and assigning the title to itself.

What you need to do here is just to check if `title` is missing and if so then you need to assign `name` to it or just an empty string. You even can keep `name` property:

```javascript
if (!title) {
  title = name ? name : ''
}
```

Reply:
@anuveyatsu will be refactored